model/report_sources/proof_of_concept_2020-03-22.Rmd

```diff
@@ -162,7 +171,7 @@ This section contain information on the various parameters. We use these
 data to generate distribution, with discretisation when needed.
 
 * **Duration of hospitalisation**:

```

Comment:
Could add the median durantion of hospitalisation and IQR to the duration of hospitalisation tab above each illustration.